test/testinput/acftbias_converter.yaml

```diff
@@ -0,0 +1,8 @@
+input coeff file: testinput/gdas.t18z.abias_air
+default predictors: &default_preds
+- constant
+- vertical_velocity
```

Comment:
per the convo here: https://github.com/JCSDA-internal/ufo/pull/2979#discussion_r1324862968 does this need to be `windUpward`?

Reply:
"windUpward" is not correct, ascentRate is not precise, as it may be descentRate, it is aircraft ascent/descent rate.

Reply:
Well @BrettHoover-NOAA 's point was the only thing in the IODA conventions is `windUpward`, so we as a group need to figure out the proper name. As I agree, wind is not the right answer here, because it is more like `aircraftVerticalVelocity`. See spreadsheet: https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/conventions/tbls.html

Reply:
I saw "aircraftVelocity" in the spreadsheet, so I'm fine with aircraftVerticalVelocity.

Reply:
and then `aircraftVerticalVelocity_order_2`?

Reply:
@CoryMartin-NOAA The predictor function automatically appends a `"_order_N" to the variable names on my side, the names in the ctest end up being pretty long: `predictor_obs_metadata_predictor_order_1_aircraftVerticalVelocity_airTemperature`, in this case.

I have code changes I can deploy that swap out `windUpward` with `aircraftVerticalVelocity`, if you want me to push them. ctest should pass.

Reply:
I'm okay with that @BrettHoover-NOAA but I would like to confirm that others are fine with the name `aircraftVerticalVelocity` before we do a further commit/update test files.